### ISSUE TITLE
Add column with score values in Review overview page

### ISSUE THREAD

**Issue by FelixPetriconi** [ACCUConf/Submission_Web_Application#90]
From my point of view, it would be helpful if the score values would be visible on the overview page. In case that this would be too much, one could drop the yes/no column (0 has the same meaning as no)

**Comment by russel** [comment]
Good point about the 0 score. I am at a "hackfest" the next two days, but will see if I can make this change.
